components/brave_wallet/browser/json_rpc_responses.idl

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+namespace types {
```

Comment:
This should be just `json_rpc_responses` because we only generate types from idl instead of mixing API interfaces